packages/vapor/resources/icons/svg/unlock-stroked-small.svg

```diff
@@ -1 +1,9 @@
-<svg viewBox="0 0 16 16" fill="none" stroke="#282829" xmlns="http://www.w3.org/2000/svg"><path d="M2.5 10C2.5 9.17157 3.17157 8.5 4 8.5H8C8.82843 8.5 9.5 9.17157 9.5 10V13C9.5 13.8284 8.82843 14.5 8 14.5H4C3.17157 14.5 2.5 13.8284 2.5 13V10Z"/><path d="M13 8V5.75C13 4.23122 11.7688 3 10.25 3V3C8.73122 3 7.5 4.23122 7.5 5.75L7.5 8" stroke-linecap="round"/><path d="M7 11C7 11.5523 6.55228 12 6 12C5.44772 12 5 11.5523 5 11C5 10.4477 5.44772 10 6 10C6.55228 10 7 10.4477 7 11Z" fill="#282829"/><path d="M5.5 11.5C5.5 11.2239 5.72386 11 6 11C6.27614 11 6.5 11.2239 6.5 11.5V12.5C6.5 12.7761 6.27614 13 6 13C5.72386 13 5.5 12.7761 5.5 12.5V11.5Z" fill="#282829"/></svg>
\ No newline at end of file
+<svg viewBox="0 0 16 16" fill="none" stroke="#282829" xmlns="http://www.w3.org/2000/svg">
+    <path d="M2.5 10C2.5 9.17157 3.17157 8.5 4 8.5H8C8.82843 8.5 9.5 9.17157 9.5 10V13C9.5 13.8284 8.82843 14.5 8 14.5H4C3.17157 14.5 2.5 13.8284 2.5 13V10Z"/>
+    <path d="M13 8V5.75C13 4.23122 11.7688 3 10.25 3V3C8.73122 3 7.5 4.23122 7.5 5.75L7.5 8" stroke-linecap="round"/>
+    <mask id="path-3-inside-1" fill="white">
```

Comment:
Hmm This svg still mixes fill and stroke 
![image](https://user-images.githubusercontent.com/52677246/122075532-256d3780-cdc8-11eb-94f1-21810ff2e0d9.png)